_includes/tailwind-styles.html

```diff
@@ -251,6 +251,102 @@
             @apply left-full ml-[2px] top-1/2 -translate-y-1/2;
         }
 
+        /*
+        Define Tables
+        */
+
+        .htk-table {
+            @apply max-w-full text-sm text-gray-700 text-center;
+        }
+
+        .htk-table-head {
+            @apply border-b-2 border-neutral-200 tracking-wider text-neutral-500;
+        }
+
+        .hkt-table-row {
+            @apply border-b border-neutral-200;
+        }
+
+        .hkt-table-row:hover {
+            @apply bg-neutral-100;
+        }
+
+        .hkt-table-row-name {
+            @apply tracking-wider text-neutral-500;
+        }
+
+        .hkt-table-footer {
+            @apply border-t border-neutral-200;
+        }
```

Comment:
These things may not even need classnames. They're default parts of tables and you are already putting `.htk-table` on the top level of them. So you can use:

- `.htk-table thead`
- `.htk-table thead tr`
- `.htk-table thead th`
- `.htk-table tbody`
- `.htk-table tbody tr`
- `.htk-table tbody th`
- `.htk-table tbody td`
- `.htk-table tfoot`
- etc.